api.php

```diff
@@ -168,6 +168,33 @@
 
 	require("scripts/pi-hole/php/customdns.php");
 }
+elseif(isset($_GET['customcname']) && $auth)
+{
+	if(isset($_GET["auth"]))
+	{
+	if($_GET["auth"] !== $pwhash)
+		die("Not authorized!");
+	}
+	else
+	{
+		// Skip token validation if explicit auth string is given
+		check_csrf($_GET['token']);
+	}
```

Comment:
```php
	if (isset($_GET["auth"])) {
		if ($_GET["auth"] !== $pwhash) {
			die("Not authorized!");
		}
	} else {
		// Skip token validation if explicit auth string is given
		check_csrf($_GET['token']);
	}
```

The indentation is strange, causing a little confusion between the two `if` statements.
I suggest a change, just for readability.

Reply:
fixed, thank you